registry.go

```diff
@@ -76,13 +76,13 @@ type ManifestService interface {
 	Exists(dgst digest.Digest) (bool, error)
 
 	// Get retrieves the identified by the digest, if it exists.
-	Get(dgst digest.Digest) (*manifest.SignedManifest, error)
+	Get(dgst digest.Digest) (*schema1.SignedManifest, error)
```

Comment:
I assume that this is an intermediate step, and eventually methods like this will use abstractions provided by the `manifest` package, right? Same question for the `notifications` package.

No objection to doing it this way; just want to confirm that's the path forward.